README.md

```diff
@@ -1 +1,241 @@
+
+# .gitignore
+node_modules/
+
+
+## Web Hint 
+This is a customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors.
+
+**NOTE:** If you are using Windows, make sure you initialize npm to create `package.json` file. 
+   ```
+   npm init -y
+   ```
+
+1. Run
+   ```
+   npm install --save-dev hint@7.x
+   ```
+   *how to use npm: (https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).*
+2. Copy [.hintrc](.hintrc) to the root directory of your project.
+3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+4. Run
+   ```
+   npx hint .
+   ```
+5. Fix validation errors.
+
+### [Stylelint](https://stylelint.io/)
+
+A mighty, modern linter that helps you avoid errors and enforce conventions in your styles.
+
+1. Run
+
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+not sure how to use npm? Read this.
+
+2. Copy .stylelintrc.json to the root directory of your project.
+
+3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+
+If you think that change is necessary - open a Pull Request in this repository and let your code reviewer know about it.
+4. Run npx stylelint "**/*.{css,scss}" on the root of your directory of your project.
+
+5. Fix linter errors.
+
+6. **IMPORTANT NOTE:** feel free to research auto-correct options for Stylelint if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+
+### Making Your HTML and CSS Code From Figma
+-Pick a figma template to work on and use css and html to build a replica the first 2 sections of the mobile website using the template you have chosen.
+
+-Make sure to replace the contents of helloworld.html with new contents of your own html code which replicates the chosen figma template. 
+
+-In your figma template, you will create the toolbar (or header) and the headline section (right after the header)
+
+**note**
+-Make sure to pick the mobile template on figma and not the desktop template.
+-In order to lay out the elements on the page you should use Flexbox in all 2 sections.
+-You don't need to implement any functionality that requires JavaScript, like opening the menu or creating dropdowns.
+-You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using the templates in Figma.
+-Implement the button interactions (enable, hover, etc.).
+
+
+
```

Comment:
- [ ] [ OPTIONAL] I would like to strongly recommend you remove all unnecessary information from your readme and keep the order from the original [readme template ](https://github.com/microverseinc/readme-template)to make it more professional :+1: